api/tests/unit/application/target-profiles/index_test.js

```diff
@@ -766,89 +766,98 @@ describe('Integration | Application | Target Profiles | Routes', function () {
     });
   });
 
-  describe('POST /api/target-profiles/{id}/badges', function () {
+  describe('POST /api/admin/target-profiles/{id}/badges', function () {
+    const method = 'POST';
+    const url = '/api/admin/target-profiles/123/badges';
+    const payload = {
+      data: {
+        type: 'badges',
+        attributes: {
+          key: 'KEY',
+          'alt-message': 'alt-message',
+          'image-url': 'https://example.net/image.svg',
+          message: 'message',
+          title: 'title',
+          'is-certifiable': false,
+          'is-always-visible': true,
+        },
+      },
+    };
+
     it('should return 201 HTTP response', async function () {
```

Comment:
ce test `it('should return 201 HTTP response'` peut être supprimé car celui qui teste avec les roles fait la même chose en plus complet